src/OV/OpenVarioMenu.cpp

```diff
@@ -391,6 +391,7 @@ Main()
 
   UI::TopWindowStyle main_style;
   main_style.Resizable();
+  main_style.InitialOrientation(Display::DetectInitialOrientation());
```

Comment:
I wish I could cherry-pick this commit, but it fails to compile. Sigh.

Reply:
It fails because ov.mk needs:
$(SRC)/Hardware/DisplayGlue.cpp \

Reply:
But that is not enough. I cherry-picked that commit and can't get it to compile either, even after adding almost the whole content of `main.mk`. The dependencies of `DisplayGlue.cpp` blow up the makefile and I wonder if there is a "cleaner" way of doing this.

Reply:
Hmmm, I can compile the Image for the OpenVario without any problems.

Reply:
But you can't compile the binary for the OpenVarioMenu standalone (try `make output/UNIX/bin/OpenVarioMenu`) and that is what's required. The dependencies are probably taken from the preceding XCSoar build.

Reply:
thank you for your explanation!!